docs/usage/using-static-ip.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 3
+---
+# About huronOS instance Static IP configuration
+This feature allows you to assign each usb a static IP address, this way, you will not have to assign an IP address each time an instance has booted up if a DHCP server is not available/enabled. 
+
+The current implementation is expected to work with too little issues, yet since testing is one of the things we lack the most, we're not sure on how smooth the use of this feature is going to be.
+
+# How to use a static IP on an instance
+
+To use static IP, you need to assign the instance ip address, ip gateway and network mask at the time of using the installer, for example `sudo ./install.sh -ia 192.168.1.2 -ig 192.168.1.254 -nm`
+
+# Notes
+
+- This feature only works on ethernet interfaces (since wifi connections are more likely to provides an IP, whereas this feature was born from the lack of dynamic ip leasing in one institution that uses HuronOS)
+
+- If the computer has multiple network interfaces it will try on each available interface in no specific order until the configuration gets internet output through said interface, if it had no internet that interface would go back to dhcp
```

Comment:
I would made a section called `## How it works` and explain what the ipman script does. 
'The system will scan all the network interfaces and select the ethernet ones to try setting the static IP address with the configured data. The first ethernet interface to set up link and access to the directives file will be the interface to be used by the system. If no interface could be configured, then the system will set all interfaces to use DHCP instead.'

Something like that would be better to have a better understanding of what the system tries to do and then users can expect a behavior on this.

Reply:
Done.